config/ses.toml

```diff
@@ -24,3 +24,6 @@ target = "devel7"
       storage        = "http://download.suse.de/ibs/SUSE/Products/Storage/7/x86_64/product/"
       storage_devel  = "http://download.suse.de/ibs/Devel:/Storage:/7.0/SLE_15_SP2"
       storage_update = "http://download.suse.de/ibs/SUSE:/SLE-15-SP2:/Update:/Products:/SES7/standard/"
+
+    [ses.devel7.registry]
```

Comment:
This probably doesn't need to be a sub-section. You can just put registry under the `[ses.devel7]` title.